src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random rand;
+
     public String getRandomColor() {
-        return null;
+        int i = rand.nextInt(Color.values().length);
```

Comment:
```suggestion
        int i = rand.nextInt(Color.values().length);
```
i is not informative name